internal/aws/iam/iam_test.go

```diff
@@ -0,0 +1,12 @@
+package iam
+
+import "testing"
+
+func TestTransformCallerArn2(t *testing.T) {
+	if TransformCallerArn("arn:aws:sts::755952356119:assumed-role/Admin/khmoryz") != "arn:aws:iam::755952356119:role/Admin" {
```

Comment:
Are you ok with having your account id and username in the code? I don't consider account ids to be sensitive information, but want to make sure.

Reply:
@ericzbeard 
Thank you for your concern.
But, `755952356119` is not my account id.

I just referenced the other code:)

```sh
$ git grep '755952356119'
README.md:      TheS3URI: s3://rain-artifacts-755952356119-us-east-1/a84b588aa54068ed4b027b6e06e5e0bb283f83cf0d5a6720002d36af2225dfc3 
docs/README.tmpl:      TheS3URI: s3://rain-artifacts-755952356119-us-east-1/a84b588aa54068ed4b027b6e06e5e0bb283f83cf0d5a6720002d36af2225dfc3 
internal/aws/iam/iam.go:// arn:aws:sts::755952356119:assumed-role/Admin/ezbeard-Isengard
internal/aws/iam/iam.go:// arn:aws:iam::755952356119:role/Admin
internal/aws/iam/iam.go:                                arn:aws:sts::755952356119:assumed-role/Admin/ezbeard-Isengard
internal/aws/iam/iam.go:                                This is the actual role: arn:aws:iam::755952356119:role/Admin
internal/aws/iam/iam_test.go:   if TransformCallerArn("arn:aws:sts::755952356119:assumed-role/Admin/khmoryz") != "arn:aws:iam::755952356119:role/Admin" {
internal/aws/iam/iam_test.go:   if TransformCallerArn("arn:aws:iam::755952356119:user/khmoryz") != "arn:aws:iam::755952356119:user/khmoryz" {
internal/cmd/forecast/forecast.go:              callerArn, err := iam.GetCallerArn(cfg) // arn:aws:iam::755952356119:role/Admin
test/templates/fmtziplines.yaml:      Role: arn:aws:iam::755952356119:role/lambda-basic
test/templates/fmtziplinesfolded.yaml:      Role: arn:aws:iam::755952356119:role/lambda-basic
test/templates/forecast-fail.yml:                - "arn:aws:iam::755952356119:role/Admin"
test/templates/forecast-lambda-pkg.yaml:        S3Bucket: rain-artifacts-755952356119-us-east-1
test/templates/forecast-succeed.yml:                - "arn:aws:iam::755952356119:role/Admin"
```